tests/evolution_test/evolution_v1_generated.h

```diff
@@ -42,24 +45,24 @@ inline const char * const *EnumNamesEnum() {
 }
 
 inline const char *EnumNameEnum(Enum e) {
-  if (e < Enum::King || e > Enum::Queen) return "";
+  if (flatbuffers::IsOutRange(e, Enum_King, Enum_Queen)) return "";
   const size_t index = static_cast<size_t>(e);
   return EnumNamesEnum()[index];
 }
 
-enum class Union : uint8_t {
-  NONE = 0,
-  TableA = 1,
-  TableB = 2,
-  MIN = NONE,
-  MAX = TableB
+enum Union {
+  Union_NONE = 0,
+  Union_TableA = 1,
+  Union_TableB = 2,
+  Union_MIN = Union_NONE,
+  Union_MAX = Union_TableB
 };
 
 inline const Union (&EnumValuesUnion())[3] {
   static const Union values[] = {
-    Union::NONE,
-    Union::TableA,
-    Union::TableB
+    Union_NONE,
```

Comment:
I honestly don't know what happened, I simply run:
```
bazel run :flatc -- --cpp -o ~/workspace/flatbuffers/tests/evolution_test ~/workspace/flatbuffers/tests/evolution_test/evolution_v1.fbs
bazel run :flatc -- --cpp -o ~/workspace/flatbuffers/tests/evolution_test ~/workspace/flatbuffers/tests/evolution_test/evolution_v2.fbs
```
and these updated generated. Should I use a different set of options?

Reply:
The default `test/generate_code.sh` script should suffice. I just checked it and the line to generated it was removed here for some reason: https://github.com/google/flatbuffers/commit/44bf719883248a392e04d9cebecdedb00cdd393c#diff-095ca9891b22d443d2181ad26d248555L22

You can use this line for now:

`flatc --cpp --scoped-enums -o evolution_test ./evolution_test/evolution_v*.fbs`

I'll look into re adding this line again.